src/Azure.php

```diff
@@ -53,6 +53,7 @@ public function handle($request, Closure $next)
                     'client_secret' => config('azure.client.secret'),
                     'refresh_token' => $refresh_token,
                     'resource' => config('azure.resource'),
+                    'redirect_uri' => route('azure.callback'),
```

Comment:
Make this an optional parameter.  Perhaps move `form_param` into its own hash above the try catch.  Then, check to see if the `route('azure.callback`)` exists.  If it does, add the key.